test/run-tests.c

```diff
@@ -210,7 +210,9 @@ static int maybe_run_test(int argc, char **argv) {
     notify_parent_process();
     ASSERT(sizeof(fd) == read(0, &fd, sizeof(fd)));
     ASSERT(fd > 2);
+# ifndef __PASE__  /* On IBMi PASE, write() returns 1 */
```

Comment:
Is there any reason not to also assert on IBMi?

Reply:
> Is there any reason not to also assert on IBMi?

added the assertion.